src/nvhttp.cpp

```diff
@@ -287,7 +287,7 @@ namespace nvhttp {
     launch_session.enable_sops = util::from_view(get_arg(args, "sops", "0"));
     launch_session.surround_info = util::from_view(get_arg(args, "surroundAudioInfo", "196610"));
     launch_session.gcmap = util::from_view(get_arg(args, "gcmap", "0"));
-    launch_session.enable_hdr = util::from_view(get_arg(args, "enableHdr", "0"));
+    launch_session.enable_hdr = util::from_view(get_arg(args, "hdrMode", "0"));
 
     uint32_t prepend_iv = util::endian::big<uint32_t>(util::from_view(get_arg(args, "rikeyid")));
```

Comment:
Can you remove the error log below while you're making changes here? I think it's a leftover debugging artifact.